frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -75,7 +75,7 @@ class DocumentList extends Component {
   constructor(props) {
     super(props)
 
-    this.state.selectedRows = {}
+    this.state = this.defaultLocalState()
```

Comment:
I see your point, however I believe it makes sense to use a method, as the default state has real potential to become pretty large and contain complex conditional parameters. This is for consistency and readability. 

Method name changed.